Adding support for Swedish

jopotts opened the pull request.

davidcelis commented on lib/inflections/sv.rb:
```diff
@@ -0,0 +1,93 @@
+    inflect.plural(/de$/i, 'der') # fiende, fiender
+
+    # -n
+    inflect.plural(/le$/i, 'lem') # muskel, muskler
```
This comment is a typo, yeah?

jopotts replied:
Well spotted. I don't speak Swedish myself, but that's no excuse for a wrong comment! I took these inflections from directly from here:

https://github.com/mynewsdesk/swedish_pluralize/blob/master/lib/swedish_pluralize.rb

I had to change the order of some of the definitions to get the tests (which I created from the trailing comments) to pass too. I've got no idea if these are correct in fact so I'll get my translator to have a look and get back to you...

fjalestad replied:
Yep, poor example. # muskeln, musklerna would be
`inflect.plural(/n$/i, 'lerna') # muskeln, musklerna`

jopotts commented:
I've fixed the comments and moved some inflections, added a couple more tests, and got the translator to check the tests and all look ok. I'm sure it's not perfect, but I think it's worth merging until we can find a Swedish speaking Ruby dev to come and improve. Thanks!
